From jic23@cam.ac.uk  Mon Oct 12 14:20:30 2009
From: Jonathan Cameron <jic23@cam.ac.uk>
Date: Mon, 12 Oct 2009 19:18:09 +0100
Subject: Staging: iio: Fix missing include <linux/sched.h>
Cc: Greg Kroah-Hartman <gregkh@suse.de>, devel@linuxdriverproject.org
Message-ID: <4AD372E1.50002@cam.ac.uk>


linux/sched.h include was removed form linux/poll.h by
commmit a99bbaf5ee6bad1aca0c88ea65ec6e5373e86184

Required for definition of TASK_INTERRUPTIBLE amongst others

From: Jonathan Cameron <jic23@cam.ac.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


--- a/drivers/staging/iio/industrialio-core.c
+++ b/drivers/staging/iio/industrialio-core.c
@@ -18,6 +18,7 @@
 #include <linux/fs.h>
 #include <linux/interrupt.h>
 #include <linux/poll.h>
+#include <linux/sched.h>
 #include <linux/cdev.h>
 #include "iio.h"
 #include "trigger_consumer.h"
-- 
1.6.3.3

